### PR TITLE
Run & load Dyon code from a String

### PR DESCRIPTION
**Pull request by ichy-wayland**
Hey! I was trying to make a dyon REPL, but there was no way to execute code directly from a String.
I hacked this in <30min, so its probably horrible, but the idea of running and loading code from a String is what matters.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -335,7 +349,7 @@ pub fn load(source: &str, module: &mut Module) -> Result<(), String> {
 
     // Convert to AST.
     let mut ignored = vec![];
-    let conv_res = ast::convert(Arc::new(source.into()), d.clone(), &data, &mut ignored, module);
+    let conv_res = ast::convert(Arc::new("".into()), Arc::new(d.clone()), &data, &mut ignored, module);
```
Comment (bvssvni) [inline src/lib.rs]: The source file must be passed when loading from a file.

Comment (ichy-wayland) [inline src/lib.rs]: agree, don't have time to fix it right now

**Comment by bvssvni** [comment]
Closed in favor of https://github.com/PistonDevelopers/dyon/pull/325

**Comment by ids1024** [comment]
> I was trying to make a dyon REPL

Did you ever achieve a working REPL?

**Pull request closed without merging**